README.rst

```diff
@@ -112,6 +112,11 @@ asset by editing the asset in Superset and selecting "Save As" to save it to a n
 # Note: If you are using custom assets you will need to rebuild your aspects-superset
 # image on your local machine with `tutor images build aspects-superset --no-cache`.
 
+Assets created for Aspects that are no longer used are specified in `aspects_asset_list.yaml`
+and will be deleted from the Superset database during `init` (specifically `import-assets`).
+Unused asset yaml files are deleted during `import_superset_zip` or by running the command
+`tutor aspects delete_unused_assets`.
```

Comment:
I think this is currently incorrect? It doesn't seem to happen on init, and the command that seems to call it is `check_superset_assets`?

Reply:
yeah i need to update it